docs/API_documentation/api_reference.md

```diff
@@ -0,0 +1 @@
+# API reference 
```

Comment:
should it be something here?

Reply:
Yes, it will be added later by the developer team.